python/python_direct/ops.cpp

```diff
@@ -1991,6 +1991,121 @@ TensorView
 )");
 }
 
+#define NVFUSER_DIRECT_BINDING_SCAN_OP(NAME, OP_NAME, OP_TYPE, DOCSTRING) \
```

Comment:
I'd prefer `NVFUSER_DIRECT_BINDING_SCAN_OP` to be placed adjacent to `NVFUSER_DIRECT_BINDING_REDUCTION_OP`